registry-scanner/hack/create-release-pr.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+### This script creates a new release PR
+# - install gh cli and semver-cli (go install github.com/davidrjonas/semver-cli@latest)
+# - create and push "release-X.Y" branch
+# - checkout this branch locally
+# - run this script from repo root: ./hack/create-release-pr.sh [REMOTE]
+# - merge the PR
+# It will trigger the release workflow that would create release draft on github
+
```

Comment:
this script takes 2 args, so the above comments need updating to reflect that. Also no PR is involved.